explore-analyze/alerts-cases/alerts/rule-type-es-query.md

```diff
@@ -54,7 +54,9 @@ When you create an {{es}} query rule, your choice of query type affects the info
     | LIMIT 10
     ```
 
-2. If you use query DSL, KQL, or Lucene, set the group and theshold.
+2. Specify details for grouping alerts.
```

Comment:
Maybe we could say something to make it more obvious that there are two options? It's a little hard to see that the time field is a different option especially with the `applies_to` tags. I'm not sure with the formatting if something like a sub unordered list would work? And the ordered list text could be something like:

```suggestion
2. Specify details for grouping alerts based on your query language.
```